docs/intro/install.md

```diff
@@ -23,6 +23,22 @@ The latest `starport` binary is downloaded from the Github repo and installed in
 
 To install previous versions of the precompiled `starport` binary or customize the installation process, see [Starport installer docs](https://github.com/allinbits/starport-installer) on GitHub.
 
+### Adding `~/go/bin` to `$PATH`
```

Comment:
Should we talk about `$GOBIN` instead? or `$GOPATH/bin` if `~/go` may not always be the `GOPATH`

Reply:
Should we be recommending adding `$GOBIN` to `$PATH` instead? Is `$GOBIN` always defined?

Reply:
both of them are optional at this point. maybe add a link to [golang documentation](https://golang.org/doc/install/source#environment) for people who are still using them.

but definitely don't recommend setting GOPATH or GOBIN as it will only confuse people

Reply:
Or giving the link to the documentation as a fallback if `$GOPATH` is not defined (which I believe is defined more often than `$GOBIN`)